boefjes/images/generic_oci_adapter.py

```diff
@@ -0,0 +1,51 @@
+import os
+from importlib import import_module
+
+import sys
+import traceback
+from base64 import b64encode
+
+import httpx
+
+
+def main():
```

Comment:
Once we iterate on this a bit, we should consider a refactor to deduplicate the code between this adapter and the regular one. The behavior is a bit different though, so for both simplicity and clarity, we chose to split them up for now